lib/lorekeeper/simple_logger.rb

```diff
@@ -43,16 +52,28 @@ def inspect
       end
     end
 
-    def exception(exception, custom_message = nil, custom_data = nil, level = :error)
-      log_level = METHOD_SEVERITY_MAP[level] || ERROR
+    # To not raise NoMethodError for the methods defined in JSONLogger
+    JSON_LOGGER_METHODS.each do |method_name|
+      define_method method_name, ->(*, &block) {}
```

Comment:
can we define them manually? We have a story to do without meta-programming entirely for reasons detailed in MCC-635260

Reply:
updated in 4d2e20dd57fe800c077dd1517bde91f0ea24bda6